.stylelintignore

```diff
@@ -1 +0,0 @@
-wordpress
```

Comment:
It now uses the recommended Stylelint config located in `@wordpress/scripts` package.

Reply:
The `wordpress` folder is **not** included in the `.stylelintignore` file:
> https://github.com/WordPress/gutenberg/blob/master/packages/scripts/config/.stylelintignore#L3

Reply:
Ha, yes it is, its added in this PR, ignore the above 😏

Reply:
The question is whether it shouldn't be:

`/wordpress` as otherwise, it might trigger at any level of nesting. I don't know really what's the best way to proceed.

Reply:
I've got an idea on resolving this for the `@wordpress/scripts` package so that the configs can be "nested" and "inherited" in a better way than currently occurs.

Happy to see this PR land and any issues arise can be fixed in subsequent PRs

Reply:
Cool, looking forward to see your proposal.